sdk-extensions/autoconfigure/README.md

```diff
@@ -72,11 +72,13 @@ The OpenTelemetry SDK can be disabled entirely. If disabled, `AutoConfiguredOpen
 
 The following configuration properties are common to all exporters:
 
-| System property       | Environment variable  | Purpose                                                                                                                    |
-|-----------------------|-----------------------|----------------------------------------------------------------------------------------------------------------------------|
-| otel.traces.exporter  | OTEL_TRACES_EXPORTER  | List of exporters to be used for tracing, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter. |
-| otel.metrics.exporter | OTEL_METRICS_EXPORTER | List of exporters to be used for metrics, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter. |
-| otel.logs.exporter    | OTEL_LOGS_EXPORTER    | List of exporters to be used for logging, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter. |
+| System property                             | Environment variable                        | Purpose                                                                                                                                                                                           |
+|---------------------------------------------|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| otel.traces.exporter                        | OTEL_TRACES_EXPORTER                        | List of exporters to be used for tracing, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter.                                                                        |
+| otel.metrics.exporter                       | OTEL_METRICS_EXPORTER                       | List of exporters to be used for metrics, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter.                                                                        |
+| otel.logs.exporter                          | OTEL_LOGS_EXPORTER                          | List of exporters to be used for logging, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter.                                                                        |
+| otel.java.experimental.exporter.memory_mode | OTEL_JAVA_EXPERIMENTAL_EXPORTER_MEMORY_MODE | If `reusable_data`, enable reusable memory mode (on exporters which support it) to reduce allocations. Default is `immutable_data`. This option is experimental and subject to change or removal. |
```

Comment:
I think this should be a little more user-friendly -- it suggests that there are several exporters that might support this, and it suggests that things that are not metrics exporters might also support this. The user would have to go scratching around to find out which exporters support this.

Can we list them explicitly here? And if not here, then somewhere else that we can link to?

Reply:
Or if there's no support yet maybe we can just say "Implementations are coming soon, check back in the near future". :)

Reply:
Good point - will list them! The exporters which adhere to this are `OtlpHttpMetricExporter`, `OtlpGrpcMetricExporter` and `PrometheusHttpServer`